detox/android/detox/src/main/java/com/wix/detox/DetoxJUnitRunner.java

```diff
@@ -47,9 +49,15 @@ private void onBeforeAppOnCreate(Application app, Bundle arguments) {
         if (DetoxInstrumentsManager.supports()) {
             final String recordingPath = arguments.getString("detoxInstrumRecPath");
             if (recordingPath != null) {
-                final long samplingInterval = Long.parseLong(
-                        arguments.getString("detoxInstrumSamplingInterval", "250")
```

Comment:
} catch (NumberFormatException e) {
                   //log that a default is being used because input is invalid, but no stack trace, you don't want that stacktrace.
```

Reply:
Thanks! Changed.